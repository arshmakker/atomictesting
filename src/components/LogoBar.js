import React from 'react'
import Logo from './Logo'

function LogoBar({
    logoText = "Ma Web",

}) {
    return (
        <>
            <div>{logoText}</div>
            <Logo />
        </>
    )
}

export default LogoBar;