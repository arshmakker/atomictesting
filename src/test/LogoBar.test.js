import React from 'react'
import LogoBar from '../components/LogoBar'
import renderer from 'react-test-renderer'

const logoText = "This is my turf"


it("Renders the Logo Bar correctly with no props ", () => {
    const tree = renderer
        .create(<LogoBar/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})
it("Renders the Logo Bar correctly with test props", () => {

    const tree = renderer
        .create(<LogoBar logoText={logoText}
             ></LogoBar>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})