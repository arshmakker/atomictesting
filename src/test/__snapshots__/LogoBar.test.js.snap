// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Logo Bar correctly with no props  1`] = `
Array [
  <div>
    Ma Web
  </div>,
  <img
    alt="unapologetic logo"
    className=""
    src="https://img.icons8.com/ios/50/000000/pick-up.png"
  />,
]
`;

exports[`Renders the Logo Bar correctly with test props 1`] = `
Array [
  <div>
    This is my turf
  </div>,
  <img
    alt="unapologetic logo"
    className=""
    src="https://img.icons8.com/ios/50/000000/pick-up.png"
  />,
]
`;
